scripts/zones/Xarcabard/npcs/Perennial_Snow.lua

```diff
@@ -25,9 +25,13 @@ function onTrigger(player,npc)
    local circleOfTime = player:getQuestStatus(JEUNO,THE_CIRCLE_OF_TIME);
 
    if (circleOfTime == QUEST_ACCEPTED and player:getVar("circleTime") == 3) then
+       -- http://ffxiclopedia.wikia.com/wiki/The_Circle_of_Time (one earth hour delay)
+       local buried = player:getVar("star_ringburied");
+       local cleansed = buried != tonumber(os.date("%H"));
```

Comment:
what if they use in same earth hour next day?
storing the buried date as seconds since epoch and subtracting that from current time to see if an hour has elapsed would be better

also, lua != == ~=